symphony/bdk/core/auth/obo_authenticator.py

```diff
@@ -117,10 +109,10 @@ class OboAuthenticatorCert(OboAuthenticator):
     """Obo authenticator Certificate implementation.
     """
 
-    def __init__(self, certificate_authenticator_api: CertificateAuthenticationApi):
+    def __init__(self, certificate_authenticator_api: CertificateAuthenticationApi, retry_config: BdkRetryConfig):
         self._authentication_api = certificate_authenticator_api
+        self._retry_config = retry_config
 
-    @wrap_api_exception
     async def retrieve_obo_session_token_by_user_id(self, user_id: int) -> str:
```

Comment:
This should be retried, esp. line 124: ` await self._authentication_api.v1_app_user_uid_authenticate_post`